posturebest/Views/ConfigurationsView.swift

```diff
@@ -1,28 +1,47 @@
-//
-//  ConfigurationsView.swift
```

Comment:
nit: why remove header here? idk how strict we need to be on formatting, but I think its good to be consistent